chainercv/links/connection/convolution_2d_block.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+
+import chainer
+from chainer.functions import relu
+from chainer.links import BatchNormalization
+from chainer.links import Convolution2D
+
+
+class Convolution2DBlock(chainer.Chain):
+    """Convolution2D --> (Batch Normalization) --> Activation
```

Comment:
Personally, I find the verbose `chainer.Chain` to be easier to read than `SequentialX`.
(I strongly believe that `SequentialFeatureExtractor` is good for implementing a feature extractor like VGG.)

Besides, I do not think that blocks need any of the features `SequentialFeatureExtractor` provides.